packages/aws-cdk-lib/aws-rds/lib/instance.ts

```diff
@@ -347,6 +347,45 @@ export enum NetworkType {
   DUAL = 'DUAL'
 }
 
+/**
+ * The CA certificate used for this DB instance.
```

Comment:
One minor nitpick… add `@see: https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/UsingWithRDS.SSL.html` in jsdocs section.